server/graphql-context.js

```diff
@@ -0,0 +1,6 @@
+const getGraphQLContext = ({ req }) => {
+    const user = req.session?.user ?? null;
+    return { user };
+};
```

Comment:
Assumes Mike will add Node 14 support. If he doesn't I'll update it to use the Node 12 syntax here.

Reply:
He wants to do it in a separate PR, not the upcoming one. So I reverted the syntax.